src/FondOfSpryker/Zed/CompanyUsersRestApi/Business/CompanyUser/CompanyUserWriter.php

```diff
@@ -170,14 +170,19 @@ public function create(
             return $this->apiError->createAccessDeniedErrorResponse();
         }
 
-        $companyBusinessUnit = $this->findDefaultCompanyBusinessUnitOf($companyTransfer);
+        $companyBusinessUnit = $this->companyBusinessUnitFacade
+            ->findDefaultBusinessUnitByCompanyId($companyTransfer->getIdCompany());
 
         if ($companyBusinessUnit === null) {
             return $this->apiError->createDefaultCompanyBusinessUnitNotFoundErrorResponse();
         }
 
         $filterCustomerTransfer = $this->createCustomerTransferFrom($restCompanyUsersRequestAttributesTransfer);
-        $customerTransfer = $this->findCustomerTransferFrom($filterCustomerTransfer);
+        $customerTransfer = $this->customerFacade->getCustomer($filterCustomerTransfer);
```

Comment:
Bridge?

Reply:
@DerCalli The Class needs to be still refractored